versioned_docs/version-v3.0.0/node/node-macos.md

```
@@ -0,0 +1,188 @@
+## Install HOPRd node
+
+1. **Install Homebrew**
+
+   1. Open the **Terminal** application on your macOS.
+   2. Install Homebrew follow this guide: [https://brew.sh](https://brew.sh/).
+
+2. **Install HOPRd using homebrew**
+
+   Install the HOPRd package on your macOS using Homebrew:
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)